src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,53 @@
-import React from 'react';
+import { useState } from 'react';
+
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { MovieCard } from '../MovieCard';
+
+import { getMovie } from '../../api';
+import { normalizeMovieData } from './helper';
+
+import { Movie } from '../../types/Movie';
+
+type FindMovieProps = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie = ({ onAdd }: FindMovieProps) => {
+  const [movieInput, setMovieInput] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [movieError, setMovieError] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = (e: React.SyntheticEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setLoading(true);
+
+    getMovie(movieInput)
+      .then(data => {
+        if (data.Response === 'True') {
```

Comment:
According to Mate guide it should be handled this way:
<img width="260" alt="image" src="https://github.com/mate-academy/react_movies-list-fetch-movies/assets/126275137/66f1f381-5d7e-40de-8bb4-a396e4639990">

But to me it also looks good